test/gtest/group_conv2d_wrw.cpp

```diff
@@ -35,3 +35,4 @@ DEFINE_GROUP_CONV2D_TEST(float, BackwardWeights);
 // DEFINE_GROUP_CONV2D_TEST(half, BackwardWeights);
```

Comment:
This looks good. Please try uncommenting `half` for backward weights to see if it's still broken.

Reply:
> This looks good. Please try uncommenting `half` for backward weights to see if it's still broken.

Hi Amber, sure. I can try to uncomments them to see if they can pass the CI.
Also, I observed some failed test cases in bwd solvers after enable the bf16 supports. Debugging now.